pkg/build/controller/strategy/custom_test.go

```diff
@@ -66,8 +66,9 @@ func TestCustomCreateBuildPod(t *testing.T) {
 	// input secret
 	// build-system-configmap
 	// container storage
```

Comment:
please add your expected mount to this list here.  it makes it easier for future people who have to touch this test.  or delete the comment, but fix up the test below that iterates the specific mounts, to make it actually iterate *all* the mounts instead of just checking a few of them.